analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -205,49 +205,42 @@ class Conan(
     }
 
     /**
-     * Return the dependency tree starting from a [rootNode] for given [scopeName].
+     * Return the dependency tree starting from a [pkgInfos] for given [scopeName].
      */
     private fun parseDependencyTree(
-        rootNode: JsonNode,
-        workingDir: File,
+        pkgInfos: JsonNode,
```

Comment:
"from a [pkgInfos]" reads a bit odd in terms of plural vs. singular. Looking at the code here and also in the caller to me singular form reads a bit better. What do you think?

Reply:
I have rewritten the function docs. For the variable I prefer plural because it contains information about all packages.